examples/thoughts/src/client/LoginPage.jsx

```diff
@@ -1,17 +1,16 @@
-import { LoginForm } from "wasp/client/auth";
-import React from 'react'
```

Comment:
The `React` import is only necessary for old versions of the JSX transform. We use Vite so it should not be necessary, that's why prettier is automatically removing it.